packages/react-ui-components/src/components/SchemaForm/Widgets/FileWidget/FileWidget.tsx

```diff
@@ -0,0 +1,196 @@
+import { EComponentSize, EActionButtonType, EIconName } from '@kelvininc/ui-components';
+import { dataURItoBlob, FormContextType, RJSFSchema, StrictRJSFSchema, WidgetProps } from '@rjsf/utils';
+
+import { KvActionButtonIcon, KvActionButtonText, KvIcon } from '../../../stencil-generated';
+import React, { ChangeEvent, useCallback, useMemo, useState } from 'react';
+import styles from './FileWidget.module.scss';
+import { get, isEmpty } from 'lodash';
+import classNames from 'classnames';
+
+function addNameToDataURL(dataURL: string, name: string) {
+	if (dataURL === null) {
+		return null;
+	}
+	return dataURL.replace(';base64', `;name=${encodeURIComponent(name)};base64`);
+}
+
+type FileInfoType = {
+	dataURL?: string | null;
+	name: string;
+	size: number;
+	type: string;
+};
+
+function processFile(file: File): Promise<FileInfoType> {
```

Comment:
Move these methods to a utils file 🙏

Reply:
and define the types in a separated file